src/main/views/user-roles/create-user-roles.html

```diff
@@ -47,14 +47,21 @@ <h2 class="heading-large padding">Create User Roles</h2>
         </div>
     {% endif %}
 
-    <form id="createuserroleform" name="createuserform" method="POST" action="/createuserrole">
+    <form id="createuserroleform" name="createuserform" method="POST" action="{{submitUserRoleEndPoint}}">
     
       {{ csrfProtection(csrfToken) }}
         <div class="form-group">
+        
             <label class="form-label" for="userroles">User Roles</label>
-            <input type="text" class="form-control" id="role" name="role" maxlength="100"/><br><br> 
+             {%if update == true%}
+                 <input type="hidden" id="role" name="role" value="{{role}}" />
+                 <label class="form-label-bold" >{{role}}</label><br><br>
+                 <input class="form-control" id="update" name="update" type="hidden" value="{{update}}"/>
+             {% else %}
```

Comment:
Slightly strange indentation here - should line up with `{% endif %}` on Line 62

Reply:
addressed